docs/CodeDevelopment/compile.md

```diff
@@ -0,0 +1,74 @@
+# How to Build/Compile
+
+The repository/product/project should describe any unique steps required.  Project Mu currently supports two methods of building.  Those will be described here to encourage pattern/code reuse and limit the required repository specific documentation.  
+
+## Compile Testing aka __MuBuild__
+
+??? danger "BETA feature"
+    This feature may change.  Build Env work is ongoing and in an effort to make more portable
+    this might change.  
+
+MuBuild is a framework for running a battery of tests against a single Mu repository (and its dependencies).  A plugin model is used for adding additional tests.  Today one such plugin is a basic compile test.  The repository maintainer may add additional tests such as linters, etc.
+
+It is often desirable to compile test code and at times there might not be a product to test with.  This is also how the Pull Requests gates are implemented and enforced.
+
+!!! Note
+    This also runs the other static code tests so it does more than compile.  
+
+    Assumption is that the repository to compile has been cloned to your filesystem and is in the state ready to compile. 
+
+1. Open cmd prompt at workspace root
+2. Run 'mu_bootstrap` to get the necessary build components
```

Comment:
Is this still accurate? Does it address any module installs that are required to get mu_bootstrap?

Reply:
prereq page has details on current suggested process

Reply:
*Here is another review but this time i used markdown* 

__does this show up as bold__